src/pages/EntityDetails/Model/Model.js

```diff
@@ -116,6 +116,7 @@ const Model = () => {
     "Cloud/Region": `${cloudProvider} / ${modelStatusData?.model.region}`,
     version: modelStatusData?.model.version,
```

Comment:
We should dynamically generate these keys based on the existence of the values so we don't have a number of empty fields in the UI if the data isn't available.